src/config.development.js

```diff
@@ -21,6 +21,7 @@ const UNLIMITED_CONTENT = UNLIMITED_CONTENT_QUERY === undefined
 ;
 
 module.exports = {
+  APP_ENV: 'development',
```

Comment:
we are loading `config.development` in `else` statement, before that we are checking for `production` and `test`
```js
if (process.env.REACT_APP_ENV === 'production') {
  Object.assign(config, require('./config.production'));
} else if (process.env.REACT_APP_ENV === 'test') {
  Object.assign(config, require('./config.test'));
} else {
  Object.assign(config, require('./config.development'));
}
```

Reply:
ugh, since we can start scripts without specifying env then it will default to `development` which is wrong because it will then cache trees..

Reply:
I've removed `APP_ENV: 'development'` from the config and set `REACT_APP_ENV=development` in `yarn start` script
so now `APP_ENV` will be `undefined` when running scripts (as it was before this change) and it will be set to `development` when running with `yarn start`